sql/chunk.sql

```diff
@@ -21,6 +21,10 @@ CREATE OR REPLACE FUNCTION _timescaledb_internal.calculate_chunk_interval(
         chunk_target_size BIGINT
 ) RETURNS BIGINT AS '@MODULE_PATHNAME@', 'ts_calculate_chunk_interval' LANGUAGE C;
 
+-- Get the status of the chunk
+CREATE OR REPLACE FUNCTION _timescaledb_internal.chunk_status(REGCLASS) RETURNS INT
```

Comment:
Some documentation of the possible status values would be nice to have either here or in the `ts_chunk_status()` definition.

Reply:
The constants are documented in `chunk.c`, but added a reference to them in the moved function.